spec/unit/puppet/provider/chocolateyfeature/windows_spec.rb

```diff
@@ -0,0 +1,170 @@
+require 'spec_helper'
+require 'stringio'
+require 'puppet/type/chocolateyfeature'
+require 'puppet/provider/chocolateyfeature/windows'
+require 'rexml/document'
+
+provider = Puppet::Type.type(:chocolateyfeature).provider(:windows)
+describe provider do
+  let (:name) { 'allowglobalconfirmation' }
+  let (:resource) { Puppet::Type.type(:chocolateyfeature).new(:provider => :windows, :name => name) }
+  let (:choco_config) { 'c:\choco.config' }
+  let (:choco_config_contents) { <<-'EOT'
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Both operate on the same xml config file, but look at different parts of it. I largely kept it the same, but can see the need for having differences in the future.